pxr/usd/plugin/usdAbc/alembicReader.cpp

```diff
@@ -894,22 +878,58 @@ _ReaderContext::_ReaderContext() :
 }
 
 bool
-_ReaderContext::Open(const std::string& filePath, std::string* errorLog)
+_ReaderContext::Open(const std::string& filePath, std::string* errorLog,
+                     const SdfFileFormat::FileFormatArguments* args)
 {
     Close();
 
-    IArchive archive;
+    std::vector<std::string> layeredABC;
+    if (args) {
+        const auto abcLayers = args->find("abcLayers");
```

Comment:
Although you could, I won't ask you to add the string-literals for the arg-strings into alembicUtils.h as Tokens (both because you only use each once, and because, since this is a plugin, clients can't really use the header files anyways).

**But**, please do add a new section to overview.dox that both names the new args, and gives example authored asset paths that use them.  And for bonus points, refer people to SdfLayer::CreateIdentifier() as a helper for building up such asset paths.  That will show up [here](http://graphics.pixar.com/usd/docs/api/usdabc_page_front.html).